MyToolsYourToolsBackend/MyToolsYourToolsBackend.Domain/Entities/User.cs

```diff
@@ -16,6 +16,10 @@ public class User : Enitity
         public IEnumerable<UserGroup> UserGroups { get; set; } = new List<UserGroup>();
         public IEnumerable<Offer> Offers { get; set; } = new List<Offer>();
         public IEnumerable<Rent> Rents { get; set; } = new List<Rent>();
+        public IEnumerable<Notification> TargetedNotifications { get; set; } = new List<Notification>();
+        public IEnumerable<Notification> SentNotifications { get; set; } = new List<Notification>();
```

Comment:
Wydaje mi się że użytkownik musi posiadać tylko swoje notyfikacje (w sensie jako ownerId) czyli te które ktoś mu nadesłał, bo te które on wysyła to już go tak jakby nie interesują. Da się to tak zrobić? W sensie po prostu wywalić to SentNotifications, czy z poziomu backendu jest to potrzebne?

Reply:
Tez mi sie wydaje ze to nie jest dla nas wogole potrzebne, ale baza to potrzebuje do konfiguracji bo inaczej sie nie da dodać notyfikacji do bazy

Reply:
To EF core wymaga

Reply:
Ok czaję, czyli o to samo chodzi przy opiniach

Reply:
Ta , ale to sie chyba przyda co nie , bo wtedy bedziemy wiedziec kogo opinia i kto ją dał